WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
```

Comment:
> The alternative design here would probably be separate columns on Rounds for each of the individual properties of these three classes.

I think that would add up to 2 columns for TimeLimit, 4 columns for Cutoff, and 5 columns for CombinedRoundCutoff, for a total of 11 new columns. It doesn't feel worth it to me.

> Things to think about is if the format of what you're serializing could ever change, how the SQL column looks to humans, and maybe unknown dangers. Not that it should be hard to convert to a non serialized format if it ever becomes needed.

This will not change without a regulations change, which could totally happen, but at least doesn't happen frequently. @Laura-O might be able to comment on if she anticipates any changes to the regulations about how time limits, combined cutoffs, or next round cutoffs work.